static/app/components/badge/groupPriority.tsx

```diff
@@ -34,6 +35,8 @@ type GroupPriorityDropdownProps = {
 type GroupPriorityBadgeProps = {
   priority: PriorityLevel;
   children?: React.ReactNode;
+  showLabel?: boolean;
+  variant?: 'default' | 'streamlined';
```

Comment:
copying the "streamlined" convention from issue details, open to better names to differentiate the two

Reply:
i'd vote for another name